src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -97,7 +97,7 @@ public function request(string $method, string $url, array $options = []): Respo
         $host = parse_url($authority, \PHP_URL_HOST);
         $proxy = $options['proxy']
             ?? ('https:' === $url['scheme'] ? $_SERVER['https_proxy'] ?? $_SERVER['HTTPS_PROXY'] ?? null : null)
-            // Ignore HTTP_PROXY except on the CLI to work around httpoxy set of vulnerabilities
+            // Ignore HTTP_PROXY except on the CLI to work around httproxy set of vulnerabilities
```

Comment:
Are you sure about this one?
https://httpoxy.org

Reply:
I suck ha. Thanks.